### ISSUE TITLE
Multiline text blocks trigger a warning

### ISSUE THREAD

**Issue by adamdelman** [Intsights/flake8-intsights#4]
The following code triggers an incorrect warning (I006 indentation should be gradually added):

```python
class A:
    def __init__(self):
        text_block = '''
this file has weird indentation.
'''
        print(text_block)
```
